db/seeds.rb

```diff
@@ -4,14 +4,21 @@
   {name: 'Accessories'}
   ])
 
+Brand.create!([
+  {name: 'Canon'},
+  {name: 'Sony'},
+  {name: 'Gitzo'},
```

Comment:
Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.